packages/mobile/src/fees/CalculateFee.tsx

```diff
@@ -85,7 +88,12 @@ function useAsyncShowError<R, Args extends any[]>(
 }
 
 const CalculateInviteFee: FunctionComponent<DispatchProps & InviteProps> = (props) => {
-  const asyncResult = useAsyncShowError(getInvitationVerificationFee, [], props.showError)
+  const asyncResult = useAsyncShowError(
+    (account: string, amount: BigNumber, comment: string) =>
```

Comment:
might be nice to give this function a name instead of inlining

Reply:
Slightly disagree on that, and it would conflict with the other handlers below. Any particular reason for wanting a name?